### DISCUSSION TITLE
Besside-ng "wi_open(): Operation not supported on socket" error message

### DISCUSSION THREAD

**Discussion by mariano-daniel** [aircrack-ng/aircrack-ng#2436] [General]
I googled to no avail and found nothing about this particular error. No matter what interface if give it, or if I'm sudo or not, it will give the same error.

Read the wiki here but found no info : https://www.aircrack-ng.org/doku.php?id=besside-ng

I am running Aircrack-ng on MacOS Catalina installed from Homebrew.

Any input would be greatly appreciated!

**Comment by Mister-X-** [comment]
Packet capture and injection is not available on MacOS (and Windows). You need Linux or BSD.
